build.sbt

```diff
@@ -41,3 +43,20 @@ site.jekyllSupport()
 ghpages.settings
 
 git.remoteRepo := "git@github.com:willb/silex.git"
+
+lazy val silex = project in file(".")
+
+lazy val spark = project.dependsOn(silex)
+  .settings(commonSettings:_*)
+  .settings(
+    name := "spark",
+    initialCommands in console := """
```

Comment:
Why not just lean on [`ReplApp`](https://github.com/willb/silex/blob/develop/src/main/scala/com/redhat/et/silex/app/app.scala#L132) here?